test/api/review/api_test.go

```diff
@@ -0,0 +1,133 @@
+package review
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestReviewCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Review CRUD", func(t *testing.T) {
+
+			// Create Application and Review.
+			app := api.Application{
+				Name:        r.Application.Name,
+				Description: "Application for Review",
+			}
+			assert.Must(t, Application.Create(&app))
+
+			// Update Application ID with the Sample Id.
+			r.Application.ID = app.ID
+
+			assert.Must(t, Review.Create(&r))
+
+			// Get.
+			got, err := Review.Get(r.ID)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+			if got.BusinessCriticality != r.BusinessCriticality {
+				t.Errorf("Different response error. Got %v, expected %v", got.BusinessCriticality, r.BusinessCriticality)
+			}
+			if got.EffortEstimate != r.EffortEstimate {
+				t.Errorf("Different response error. Got %v, expected %v", got.EffortEstimate, r.EffortEstimate)
+			}
+			if got.ProposedAction != r.ProposedAction {
+				t.Errorf("Different response error. Got %v, expected %v", got.ProposedAction, r.ProposedAction)
+			}
+			if got.WorkPriority != r.WorkPriority {
+				t.Errorf("Different response error. Got %v, expected %v", got.WorkPriority, r.WorkPriority)
+			}
+			if got.Comments != r.Comments {
+				t.Errorf("Different response error. Got %v, expected %v", got.Comments, r.Comments)
+			}
+			if got.Application.Name != r.Application.Name {
+				t.Errorf("Different response error. Got %v, expected %v", got.Application.Name, r.Application.Name)
+			}
+
+			// Update Application Name.
+			r.Application.Name = "Updated " + r.Application.Name
+			assert.Should(t, Review.Update(&r))
+
+			// Find Review and check its parameters with the got(On Updation).
+			got, err = Review.Get(r.ID)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+			if got.BusinessCriticality != r.BusinessCriticality {
```

Comment:
It might make sense extract conditions below after update and after create above to a separate method something like AsserEqualReview(t *testing.T, got, expected api.review) (all fields except Ref to Application) instead of have two sets of mostly same assertions.